pkg/kubectl/cmd/portforward.go

```diff
@@ -51,15 +52,32 @@ type PortForwardOptions struct {
 }
 
 var (
+	portforwardLong = templates.LongDesc(i18n.T(`
+                Forward one or more local ports to a pod.
+
+                Use resource type/name such as svc/mysvc to select a pod. Resource type defaults to 'pod' if omitted.
+
+                If there are multiple pods matching the criteria, a pod will be selected automatically. The
+                forwarding session ends when the selected pod terminates, and rerun of the command is needed
+                to resume forwarding.`))
+
 	portforwardExample = templates.Examples(i18n.T(`
 		# Listen on ports 5000 and 6000 locally, forwarding data to/from ports 5000 and 6000 in the pod
-		kubectl port-forward mypod 5000 6000
+		kubectl port-forward pod/mypod 5000 6000
+
+		# Listen on ports 5000 and 6000 locally, forwarding data to/from ports 5000 and 6000 in a pod selected by the service
+		kubectl port-forward svc/mysvc 5000 6000
```

Comment:
oops, didn't notice `svc` was still in the sample usage

Reply:
I can fix this if #59809 is going to need more iterations.

Reply:
Never mind, simple fix, pushed.